.github/workflows/release.yml

```diff
@@ -44,13 +44,14 @@ jobs:
     - name: Test and Build
       run: |
         GIT_TAG="$(echo $GITHUB_REF | cut -d'/' -f3)"
-        ./check-coverage.sh 80
-        bash test-pysysl.sh
+        bash scripts/check-coverage.sh 80
+        bash scripts/test-pysysl.sh
+        gradle test -b test/java/build.gradle
```

Comment:
Could we merely depend on the other workflows rather than rerunning test?

Reply:
The reason I did this is GHAs run in parallel and I couldn't find a way to wait for other runs are completed.
[jobs.<job_id>.needs](https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idneeds) works only if all the jobs are in same action script.

I did not want to publish release artifacts while other runs are failing. That's why I included tests in release script. During a release, only this action script will be executed.